wiki/Rules/Content_usage_permissions/en.md

```diff
@@ -431,8 +431,10 @@ This section was last updated on March 30, 2025.
 | [![][FA]](https://osu.ppy.sh/beatmaps/artists/466) | [ak+q](https://osu.ppy.sh/beatmaps/artists/466) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
 | [![][FA]](https://osu.ppy.sh/beatmaps/artists/51) | [Akira Complex](https://osu.ppy.sh/beatmaps/artists/51) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
 | [![][FA]](https://osu.ppy.sh/beatmaps/artists/319) | [HoneyComeBear](https://osu.ppy.sh/beatmaps/artists/319) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
+|  | Igorrr | ![][partial] | Only collaborative tracks on [Ruby My Dear](https://osu.ppy.sh/beatmaps/artists/370)'s Featured Artist listing may be used. All other tracks cannot be used. |
```

Comment:
```suggestion
|  | Igorrr | ![][partial] | Only collaborative tracks on [Ruby My Dear](https://osu.ppy.sh/beatmaps/artists/370)'s Featured Artist listing may be used. Do not use or upload any other tracks otherwise. |
```

Reply:
might be better to word this as such maybe? (to fit with the rest of the page)

Reply:
i don't like "otherwise" right after "other" but i'll adjust the rest